scipp/whats-new-0.5.0.ipynb

```diff
@@ -0,0 +1,342 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run logs now stored natively\n",
+    "\n",
+    "- Previously we stored Mantid's `Run` and `Sample` as a scalar variable holding the Mantid object as a `PyObject`.\n",
+    "- `Run` is now converted into individual attributes.\n",
+    "- `Sample` still `PyObject`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import scipp as sc"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filename = 'PG3_4844_event'\n",
+    "data = sc.neutron.load(filename=f'{filename}.nxs')\n",
+    "data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Many time-dependent logs are just data arrays, so we can access, modify, plot, ... them.\n",
+    "Remember that the `value` property has to be used to access the only value of a scalar variable:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sc.plot.plot(data.attrs['proton_charge'].value, marker='.')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Improved instrument view\n",
+    "\n",
+    "- Align camera to axis buttons\n",
+    "- Cut surface tools\n",
+    "- Bug fixes, such as pixel sizes on high-res displays"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sc.neutron.instrument_view(data)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Improved plotting\n",
+    "\n",
+    "- `logx`, `logy`, `log`, and transpose buttons in toolbar\n",
+    "- now available as method in addition to free function, `data.plot()` instead of `sc.plot.plot(data)`\n",
+    "- profile plot/picking for 3-D (and higher) data\n",
+    "- Cleaned up widgets for 3-D (and higher) data\n",
+    "- Performance improvements\n",
+    "- Dynamic event data resampling (see below)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data.plot()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "time = data.bins.data.coords['pulse-time']\n",
+    "start = sc.min(time).value\n",
+    "stop = sc.max(time).value\n",
+    "time_edges = sc.array(dims=['pulse-time'], unit=time.unit, values=np.linspace(start, stop, num=100))\n",
+    "tof_edges = sc.array(dims=['tof'], unit=sc.units.us, values=np.linspace(0,17000, num=100))\n",
+    "binned = sc.bin(data, edges=[time_edges])\n",
```

Comment:
You mention this before the events section below. Also I think it would be worth mentioning bin separately as quite a lot of work has gone into that including the bin operations.

Reply:
Binned data is so much to talk about that I will just use our docs pages, I think.